packages/migrate/src/__tests__/MigrateDiff.test.ts

```diff
@@ -197,7 +134,7 @@ describe('migrate diff', () => {
         'file:dev.db',
         '--script',
       ])
-      await expect(result).rejects.toMatchSnapshot()
+      await expect(result).rejects.toThrow()
```

Comment:
Why this change?

Reply:
Like said above, relying on the full output for failures makes it hard to edit the help output of the command.
It's better to only match part of the output if we want

Reply:
See new commit [`07a9aa7` (#23715)](https://github.com/prisma/prisma/pull/23715/commits/07a9aa769b13acc04b34f392dff6e1e7a5fb414d)